cli/cmd/generate_support_archive.go

```diff
@@ -156,6 +195,75 @@ keptn generate support-archive --dir=/some/directory`,
 	},
 }
 
+func isConfiguredIngressGatewayAvailable() *errorableBoolResult {
+	res, err := exechelper.ExecuteCommand("kubectl", "get cm -n keptn ingress-config -ojsonpath='{.data.ingress_gateway}'")
+	if err != nil {
+		return newErrorableBoolResult(false, err)
+	}
+	var gatewayFullName string
+	if res == "" {
+		gatewayFullName = "public-gateway.istio-system"
+	} else {
+		gatewayFullName = res
+	}
+
+	var gatewayName, gatewayNamespace string
+	gwSplit := strings.Split(gatewayFullName, ".")
+
+	gatewayName = gwSplit[0]
+	if len(gwSplit) > 1 {
+		gatewayNamespace = gwSplit[1]
+	} else {
+		gatewayNamespace = "istio-system"
```

Comment:
If no namespace is specified, the gateway is located in the `keptn` namespace.

Reply:
good catch, I have just corrected that

Reply:
Thanks!